apps/builder/tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "**/*.tsx",
     "../../@types/**/*.d.ts"
   ],
+  "exclude": ["node_modules", "lostpixel.config.ts"],
```

Comment:
Maybe fix errors instead of suppressing them?

Reply:
The error is that `lostpixel.config.ts` should be under `rootDir`.
![image](https://user-images.githubusercontent.com/5845193/224678651-fb54690a-46ea-427c-b0de-05077396d384.png)

Should I just move it there? Or what is the suggested fix for this?

Reply:
I only excluded `lostpixel.config.ts`. Locally `pnpm checks` ends with a LOT more errors if I do not specify `node_modules` as well.

Is it good now?